frontend/src/Components/HomePage/UpcomingTrips/TripDetails.jsx

```diff
@@ -50,6 +52,12 @@ const getUserLocation = useCallback(() => {
     });
 }, []);
 
+// Generate Random seat
+const generateRandomSeat = () => {
```

Comment:
For random seat, I'd prefer us to store it in the database when we create the trip. You also need to make sure you get unique random seat for the same flight.

Reply:
Fixed it!